.github/auto_assign.yml

```diff
@@ -0,0 +1,33 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to true to add assignees to pull requests
+addAssignees: false
+
+# A list of reviewers to be added to pull requests (GitHub user name)
+reviewers:
+  - hackaugusto
+  - LefterisJP
+  - rakanalh
+  - palango
+  - jomuel
+  - karlb
+  - lullis
+#  - konradkonrad
+
+# A number of reviewers added to the pull request
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 1
+
+# A list of assignees, overrides reviewers if set
+# assignees:
+#   - assigneeA
+
+# A number of assignees to add to the pull request
+# Set to 0 to add all of the assignees.
+# Uses numberOfReviewers if unset.
+# numberOfAssignees: 2
+
+# A list of keywords to be skipped the process that add reviewers if pull requests include it
+# skipKeywords:
```

Comment:
Looks like this line needs to be de-commented

Reply:
Well spotted! Fixed.